### ISSUE TITLE
Free plastic bug

### ISSUE THREAD

**Issue by nlades** [Buuz135/Industrial-Foregoing#388]
Industrial Foregoing Version: 1.12.2-1.10.3-181
TeslaCoreLib Version: 1.12.2-1.0.15
Crashlog (If Any): 

Just as I said in the title. It's been a while since my last game of minecraft, and this time, I found a strange bug with my first crafting table. At first everything goes fine, but as soon as I drop any item in the grid and take it back to inventory, the table outputs 8 chunks of plastic. If I'm not mistaken, the original recipe for plastic was resin dust, not "void" :S

Basically, whenever i put an item in the grid and take it back again, it "crafts" 8 plastics out of thin air. The 8 plastic chunks are obtainable and the bug is reset when you exit the crafting gui.

**Comment by Buuz135** [comment]
Nothing I can do, just a classic inventory desync
